### PR TITLE
Fix code review comments

### PR DESCRIPTION
**Pull request by mszuyev**
# Related Github tickets

- https://github.com/VolumeFi/paloma/issues/9

# Background

Fixed some review comments on the Bash script

# Testing completed

- [x] checked that the `make lint` works

### CODE REVIEW

**Review by taariq — APPROVED** [review]
LGTM!

**Review by measure-fi — APPROVED** [review]
Would prefer other comment changes were also here but these were the main ones.

**Review by measure-fi — APPROVED** [review]
Would prefer other comment changes were also here but these were the main ones.

File: scripts/golangci-lint.sh [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
-  if ! command -v "$1" >>/dev/null 2>/dev/null; then
-    echo -e "\033[0;33m$1\033[0m command is not available - please install it first"
-    exit 1
-  fi
-}
-
 # installs the golangci-lint of the specific version to the specific directory
 function install_golangci_lint() {
   local version="$1"
+  local project_dir="." # cd "$(git rev-parse --show-toplevel)"
```
Comment (measure-fi) [inline scripts/golangci-lint.sh:7]: 🤨

**Pull request merged**